test/models/form_answer_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class FormAnswerTest < ActiveSupport::TestCase
+  context "associations" do
+    should belong_to(:user)
```

Comment:
Per above, please update.

Reply:
Will do!
We still want to keep the `user_id` reference within the `form_answers` table though yeah?

Reply:
We now have a person model for things like this, keeping User just for authentication and authorization. We can fully remove all refs to User on FormAnswer because FormAnswer belongs to FormSubmission which belongs to Person. So we can add a has through association on Person and FormAnswer (through FormSubmission) but remove the table column and index for user as it’s not required. Does that make sense?

Reply:
If you need more clarification let me know and I can make a look. Typing on my phone sucks haha

Reply:
Makes sense! Thank you 💯

Reply:
I added the desired changes in my most recent push! 
One issue though - I could not add a  `belongs_to :person, through :form_submission`. Turns out the `.through` option is only available with `has_many` or `has_one` associations. It doesn't work with `belongs_to` associations. Something I didn't know! 

I added a Foreign Key reference to `Person` instead to satisfy the requirement for the `belongs_to :person` association in `FormAnswer`. Is there another way we should go about this?